src/app/shared/form/builder/ds-dynamic-form-ui/relation-lookup-modal/search-tab/dynamic-lookup-relation-search-tab.component.ts

```diff
@@ -0,0 +1,158 @@
+import { Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { SEARCH_CONFIG_SERVICE } from '../../../../../../+my-dspace-page/my-dspace-page.component';
+import { SearchConfigurationService } from '../../../../../../core/shared/search/search-configuration.service';
+import { Item } from '../../../../../../core/shared/item.model';
+import { PaginatedSearchOptions } from '../../../../../search/paginated-search-options.model';
+import { SearchResult } from '../../../../../search/search-result.model';
+import { PaginatedList } from '../../../../../../core/data/paginated-list';
+import { RemoteData } from '../../../../../../core/data/remote-data';
+import { Observable, ReplaySubject } from 'rxjs';
+import { RelationshipOptions } from '../../../models/relationship-options.model';
+import { PaginationComponentOptions } from '../../../../../pagination/pagination-component-options.model';
+import { ListableObject } from '../../../../../object-collection/shared/listable-object.model';
+import { SearchService } from '../../../../../../core/shared/search/search.service';
+import { Router } from '@angular/router';
+import { SelectableListService } from '../../../../../object-list/selectable-list/selectable-list.service';
+import { hasValue, isNotEmpty } from '../../../../../empty.util';
+import { concat, map, multicast, switchMap, take, takeWhile, tap } from 'rxjs/operators';
+import { DSpaceObject } from '../../../../../../core/shared/dspace-object.model';
+import { getSucceededRemoteData } from '../../../../../../core/shared/operators';
+import { RouteService } from '../../../../../../core/services/route.service';
+import { CollectionElementLinkType } from '../../../../../object-collection/collection-element-link.type';
+import { Context } from '../../../../../../core/shared/context.model';
+
+@Component({
+  selector: 'ds-dynamic-lookup-relation-search-tab',
+  styleUrls: ['./dynamic-lookup-relation-search-tab.component.scss'],
+  templateUrl: './dynamic-lookup-relation-search-tab.component.html',
+  providers: [
+    {
+      provide: SEARCH_CONFIG_SERVICE,
+      useClass: SearchConfigurationService
+    }
+  ]
+})
+
+export class DsDynamicLookupRelationSearchTabComponent implements OnInit, OnDestroy {
```

Comment:
Please add TypeDocs to this class and its methods

Reply:
Added typedoc